### PR TITLE
feat(modules/lib): qbx.array.contains to check if an array contains a given value

### PR DESCRIPTION
**Pull request by Manason**
Uses iterative for loop for better performance compared to ox_lib's table.contains

### CODE REVIEW

**Review by D4isDAVID — CHANGES_REQUESTED** [review]
I don't believe we define `qbx.array = {}` anywhere yet. Either add that definition at the top or change the function to be something like `qbx.table.arrayContains`.

**Comment by Manason** [comment]
> I don't believe we define `qbx.array = {}` anywhere yet. Either add that definition at the top or change the function to be something like `qbx.table.arrayContains`.

Done.

File: modules/lib.lua [hunk 1]
```diff
@@ -147,6 +148,20 @@ function qbx.table.mapBySubfield(tble, subfield)
     return map
 end
 
+---Returns true if the given array contains the given value.
+---@param arr any[]
+---@param val any
```
Comment (D4isDAVID) [inline modules/lib.lua:153]: Maybe consider adding a generic here instead of `any`.

**Review by D4isDAVID — APPROVED** [review]

File: modules/lib.lua (hunk 1 above)
Comment (Manason) [reply modules/lib.lua]: Good call, done.

**Review by mafewtm — APPROVED** [review]

**Pull request merged**